### DISCUSSION TITLE
Which do you use burn tool about MINIX3?

### DISCUSSION THREAD

**Discussion by hangshan-boop** [community/community#140493] [Programming Help]
### Body

I used three burn tools. But the computer didn't boot from CD-ROM. 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hangshan-boop** [comment]
Please download Fedora Writer.

**Comment by kenil-gopani** [comment]
If your computer isn't booting from the MINIX3 CD-ROM, here are a few things to check:

### ✅ Recommended Burn Tools

Make sure you're using a reliable ISO burning tool. Here are three commonly used ones that work well with MINIX3:

* **Rufus** (Windows) – [https://rufus.ie/](https://rufus.ie/)
* **balenaEtcher** (Cross-platform) – [https://www.balena.io/etcher/](https://www.balena.io/etcher/)
* **Brasero** or **K3b** (Linux)

Make sure you're choosing "Burn image" or "Write ISO to disk" mode—not just copying the file to the disc.

### 🔍 Troubleshooting Tips

1. **Verify ISO**: Make sure the MINIX3 ISO is not corrupted (check the SHA256 if available).
2. **BIOS Settings**: Confirm your system's BIOS/UEFI is set to boot from the CD-ROM or USB drive.
3. **Legacy vs UEFI Mode**: MINIX3 typically works best with **Legacy (CSM) boot mode**.
4. **Try USB Instead**: If CD-ROM fails, use Rufus or balenaEtcher to burn MINIX3 to a USB stick and try booting from that.

If you still can't boot, please share:

* Your exact burn tools
* Whether you're burning to CD/DVD or USB
* Your system model and BIOS settings

Happy to help further with more details!

### Please mark as a answered
